arch/arm64/core/Kconfig

```diff
@@ -1,6 +1,7 @@
 # ARM64 core configuration options
 
 # Copyright (c) 2019 Carlo Caione <ccaione@baylibre.com>
+# Copyright (c) 2022 Qualcomm Innovation Center, Inc.
```

Comment:
nit: do not modify copyright for small changes to the files otherwise in no time we will have a looooong list.

Reply:
> The PR is fine but there is no board / soc actually using the A55?

The board and soc are being internally developed and the hardware is not yet publicly available.

Reply:
> nit: do not modify copyright for small changes to the files otherwise in no time we will have a looooong list.

Will need to talk to Legal on this.  Is there a guideline you can point me to (that I can point them to)?

Reply:
> The board and soc are being internally developed and the hardware is not yet publicly available.

Gotcha. But any reason to upstream only this snippet before anything else? I'm not against this, I'm just wondering whether we want to have this in together with something actually using it.

> Will need to talk to Legal on this. Is there a guideline you can point me to (that I can point them to)?

Tell them to compare the copyright of files with `git blame` ;) We would end up with a long list of names for each single file.

Jokes aside maybe [0]?

[0] https://www.linuxfoundation.org/blog/copyright-notices-in-open-source-software-projects/

Reply:
> Gotcha. But any reason to upstream only this snippet before anything else? I'm not against this, I'm just wondering whether we want to have this in together with something actually using it.

The CPU support is foundational for our internal development which for new silicon can take a while before the soc using this core is publicly available.  We could choose to enable a qemu_a55 as well but I think would provide little value over qemu_a53.  This at least provides a benefit to any other user that is also developing an A55-based solution since it indicates that we have validated Zephyr on A55.

Reply:
ok, I'm not strongly opinionated on this.

Reply:
> Tell them to compare the copyright of files with `git blame` ;) We would end up with a long list of names for each single file.

Ok, sync'ed with legal and problem solved.  Removed the copyright change.